scheduler/test/cook/test/mesos/api.clj

```diff
@@ -1216,4 +1220,43 @@
                (api/retrieve-url-path target-framework-id agent-hostname "executor-102")))
         (is (nil? (api/retrieve-url-path target-framework-id agent-hostname "executor-103")))
         (is (= (str "http://" agent-hostname ":5051/files/read.json?path=%2Fpath%2Ffor%2Fexecutor-111")
-               (api/retrieve-url-path target-framework-id agent-hostname "executor-111")))))))
\ No newline at end of file
+               (api/retrieve-url-path target-framework-id agent-hostname "executor-111")))))))
+
+(deftest test-instance-progress
```

Comment:
Both tests rely on reading/writing a common field in the db. I think that if we change how we write, then the unit test that does the assertions on the write should be the one to fail?

Perhaps we need an additional separate integration test for this whole scenario?